### ISSUE TITLE
Braintree_Instance::__toString() fatal error

### ISSUE THREAD

**Issue by bkdotcom** [braintree/braintree_php#71]
> braintree-php-2.37.0\lib\Braintree\Util.php (line 220): Object of class DateTime could not be converted to string

(`createdAt` and `updatedAt` are dateTime objects)

I get the error when debugging the return object from `Braintree_Transaction::sale()

see also https://github.com/braintree/braintree_php/issues/1`

simple fix:
```
                if ($value instanceof DateTime) {
                    $value =  $value->format('r');
                }
                $tmpArray[] = $key . $separator . $value;

```

**Comment by matthewpatterson** [comment]
Hey @bkdotcom, thanks for bringing this up! We're working on a fix for this; I'll let you know when it's released.

**Comment by matthewpatterson** [comment]
This is fixed in [3.0.0](https://github.com/braintree/braintree_php/releases/tag/3.0.0)
